src/lib/components/DayOpenWater.svelte

```diff
@@ -201,6 +219,21 @@
 {#if Settings.getOpenForBusiness(dtToLDS($viewedDate)) === false}
 	<div class="font-semibold text-3xl text-center">Closed</div>
 {:else}
```

Comment:
I can't remember all of the details, but when I first wrote this, I tried to use a table instead of the current approach, but eventually I decided that the table didn't do what I wanted it to do.  I think it was something to do with not being able to control the column width precisely enough.  It's possible, though, that I misunderstood something.

Reply:
i reviewed it can be simplified i just need to fully understand the data sources.